src/js/components/sharedComponents/commonCards/CardButton.jsx

```diff
@@ -9,6 +9,7 @@ import { Link } from "react-router-dom";
 
 const propTypes = {
     link: PropTypes.string,
+    apiLink: PropTypes.bool,
```

Comment:
this could be named something like "externalLink" to make it more generic and reusable

Reply:
renamed govLink